bedrock/firefox/views.py

```diff
@@ -337,11 +337,15 @@ def all_downloads(request, channel):
 
 @never_cache
 def firefox_os_geo_redirect(request):
-    country = get_country_from_request(request)
-    version = settings.FIREFOX_OS_COUNTRY_VERSIONS.get(
-        country,
-        settings.FIREFOX_OS_COUNTRY_VERSIONS['default']
-    )
+    locale = l10n_utils.get_locale(request)
+    if locale == 'en-US':
```

Comment:
please change this to 
```python
if locale == 'en-US' and waffle.switch('firefox-os-2.5'):
```
waffle is already imported near the top of this file, so that should be all we need, as discussed in the PR triage meeting.